e2e/README.md

```diff
@@ -55,6 +55,25 @@ npm run cypress:run
 You can test against dev/acceptance environments as well. Just be
 sure to leave off the trailing `/`.
 
+## Writing Cypress Tests
+
+This [Cypress guide](https://docs.cypress.io/guides/getting-started/writing-your-first-test.html#Add-a-test-file)
+provides an introduction on writing your first Cypress test.
+
+Automate's Cypress tests are divided into two main categories: UI and API tests,
+under the `api` and `ui` directories respectively.
```

Comment:
nitpick: The "respectively" here is saying that the UI tests are in `api` and the API tests are in `ui`.

Reply:
oh no 🙃